src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -64,11 +64,10 @@ public class StorageConfiguration : ConfigurationCategory
         [ConfigurationEntry("Storage.PrefetchResetThresholdInGb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public Size PrefetchResetThreshold { get; set; }
 
-        [Description("Size of journal files in megabytes for each storage, before forcing a sync and removing the journal files.")]
-        [DefaultValue(1024)]
-        [SizeUnit(SizeUnit.Megabytes)]
-        [ConfigurationEntry("Storage.JournalsSizeThresholdInMb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
-        public Size JournalsSizeThreshold { get; set; }
+        [Description("Number of Journals files for each storage, before forcing a sync and removing unused journal files.")]
+        [DefaultValue(2)]
+        [ConfigurationEntry("Storage.JournalsFilesAmountThreshold", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public int JournalsFilesAmountThreshold { get; set; }
```

Comment:
Amount -> Count ? or NumberOfJournalsThreshold ?

Reply:
Maybe in order to underline it applies to sync operation add Sync prefix - SyncJournalsCountThreshold ?